fundingmanager.go

```diff
@@ -684,6 +719,12 @@ func (f *fundingManager) initFundingWorkflow(targetPeer *peer, req *openChanReq)
 // wallet, then sends a funding request to the remote peer kicking off the
 // funding workflow.
 func (f *fundingManager) handleInitFundingMsg(msg *initFundingMsg) {
+	if !f.checkMaxPending(msg.peer) {
+		msg.err <- grpc.Errorf(OpenChannelFundingError, "Number of " +
```

Comment:
In case of using `grpc.Errorf` we may introduced new error code `OpenChannelFundingError` which helps to determine whether we receive exactly what expected to or not. I think it might be quite useful in near future when will want to distinguish different type of error on `RPC` client side.

```
	_, err = net.OpenChannel(ctx, net.Alice, carol, amount, 1)
	if err == nil {
		ct.Errorf("error wasn't received")
	} else if grpc.Code(err) != OpenChannelFundingError {
		ct.Errorf("not expected error was received : %v", err)
	}
```